data_model/src/permissions.rs

```diff
@@ -75,9 +79,32 @@ impl Registered for PermissionTokenDefinition {
 }
 
 impl PermissionTokenDefinition {
-    /// Construct new `PermissionTokenDefinition`
-    pub fn new(id: Id) -> Self {
-        Self { id }
+    /// Construct new [`PermissionTokenDefinition`]
+    #[inline]
+    pub fn new(id: <PermissionTokenDefinition as Identifiable>::Id) -> Self {
+        Self {
+            id,
+            params: btree_map::BTreeMap::default(),
```

Comment:
```suggestion
            params: btree_map::BTreeMap::new(),
```

Both a bit more explicit about what we want and why (an empty `BtreeMap`), more consistent (we do `Vec::new` vs `Vec::default()`), and shorter.